modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

```diff
@@ -4,17 +4,18 @@ import {Meta} from '@storybook/addon-docs/blocks';
 
 # Canvas Kit 5.0 Migration Guide
 
-Below are the breaking changes made in Canvas Kit v5. Please reach out if you have any questions
-about the update.
+Below are the breaking changes made in Canvas Kit v5. Please
+[reach out](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if you have
+any questions about the update.
 
+- [Codemod](#codemod)
 - [General Changes](#general-changes)
-  - [Canvas Kit CSS Maintenance Mode](#canvas-kit-css-maintenance-mode)
-  - [Codemod](#codemod)
-  - [Rename Core and Labs Core Modules to Tokens](#rename-core-and-labs-core-modules-to-tokens)
+  - [Compound Components](#compound-components)
   - [Slash Imports](#slash-imports)
+  - [Canvas Kit Preview & Component Promotions](#canvas-kit-preview--component-promotions)
+  - [Rename Core and Labs Core Modules to Tokens](#rename-core-and-labs-core-modules-to-tokens)
   - [Space Tokens](#space-tokens)
-  - [Compound Components](#compound-components) -
-    [Canvas Kit Preview & Component Promotions](#canvas-kit-preview-component-promotions)
+  - [Canvas Kit CSS Maintenance Mode](#canvas-kit-css-maintenance-mode)
```

Comment:
I pulled `Codemod` out into its own top-level section at the beginning of the guide since it seemed like helpful context to have right from the start.

I also re-ordered the sections within `General Changes` to call out what I thought were more wide-reaching changes at the beginning and to do a better job of introducing concepts before we reference them (like explaining slash imports before we reference the new import syntax in code snippets).